docs/pages/utilities/color.md

```diff
@@ -0,0 +1,47 @@
+# color
+
+A utility function to apply text and background color to a component.
+
+## Syntax
+
+```js
+color(props)
+```
+
+## Usage
+
+```jsx
+const Box = styled.div`
+  ${color}
+`
+```
+
+## Arguments
+
+| Argument | Type   | Description                                                     |
+| -------- | ------ | --------------------------------------------------------------- |
+| `props`  | Object | The props passed to the component, must contain a `theme` prop. |
+
+## Example
+
+```jsx
+import { color } from 'pcln-design-system'
+
+const Box = styled.div`
+  ${color}
+`
+```
+
+You can add the utility to a component to add support for a text color and a background color.
+
+```.jsx
+<Box color="primary">Primary color</Box>
+```
+
+Adding a `color` prop to your component will set the background color to that color and set the text color to either `text.lightest` or `text.base` depending on the contrast ratio with the background color.
+
+```.jsx
+<Box color="primary" bg="background">Primary on background color</Box>
+```
+
+You can also supply a `bg` prop to force a specified text color and background color, ignoring the recommended text color.
```

Comment:
It might be worth nothing that the `bg` prop is marked as deprecated in DS v3.

Reply:
Have we vetted out how removing the `bg` prop will affect our applications? I know that it is used a ton, but I do agree that most of the current use should just be replace by `color` only.

Reply:
the plan is just to mark it deprecated, not to disable it until v4 for exactly that reason